### PR TITLE
Feature change profile pic 160686906

### PR DESCRIPTION
**Pull request by manuel-saravia**
This branch contains the frontend implementation of the user strory [Change profile pic](https://www.pivotaltracker.com/story/show/160686906). It uses the standard html file selector (doesn't look too good but works, might change it) and a UI blocking component so the user can't interact with the app while the image is being uploaded. 

Also changed bound functions to  arrow functions.

### CODE REVIEW

File: src/components/EditProfile.jsx [hunk 1]
```diff
@@ -230,6 +232,7 @@ class EditProfile extends Component {
     const { history } = this.props;
 
     if (!validFields.includes(false)) {
+      this.setState({ blocking: true, messageWhenBlocked: 'Enviando información, por favor aguarde.' });
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: this should be in English

File: src/components/EditProfile.jsx [hunk 2]
```diff
@@ -251,13 +255,32 @@ class EditProfile extends Component {
     }
   }
 
+  handleImageSelection = (event) => {
+    this.setState({ blocking: true, messageWhenBlocked: 'Cargando imagen, por favor aguarde.' });
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: this should be in English

File: src/components/EditProfile.jsx [hunk 3]
```diff
@@ -303,14 +327,29 @@ class EditProfile extends Component {
                         fields={fields}
                         errors={errors}
                         params={{
-                          fieldName: 'URL de la nueva imagen',
+                          fieldName: 'Ingrese la URL de la nueva imagen',
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: this should be in English

File: src/components/EditProfile.jsx [hunk 4]
```diff
@@ -303,14 +327,29 @@ class EditProfile extends Component {
...
                           key: 'image',
-                          validPattern: 'https?://.+',
+                          validPattern: 'https?://.+|data:image.+',
                         }}
                         changeHandler={this.handleChange}
                         maxLen="200"
                       />
+                      <div className="form-group">
+                        <label htmlFor="image-loader" id="image-loader-label" className="label">
+                          o seleccione una imagen:
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: this should be in English

File: src/components/EditProfile.jsx [hunk 5]
```diff
@@ -303,14 +327,29 @@ class EditProfile extends Component {
...
+                          <br />
+                          <input
+                            id="image-loader"
+                            type="file"
+                            accept=".png, .jpg, .jpeg, .gif"
+                            onChange={this.handleImageSelection}
+                          />
+                        </label>
+                        <small className="form-text text-muted">
+                            También puede arrastrar y soltar una imagen sobre el selector.
```
Comment (rocespedes) [inline src/components/EditProfile.jsx]: this should be in English

**Comment by manuel-saravia** [comment]
> Some comments

@rocespedes made those changes you asked for.

**Pull request merged**